core/src/main/scala/stryker4s/extension/mutationtype/ConditionalExpression.scala

```diff
@@ -0,0 +1,13 @@
+package stryker4s.extension.mutationtype
+
+import scala.meta.Term
+import stryker4s.extension.TreeExtensions.IsConditionOfIf
+
+case object IfStatementCondition {
+
+  def unapply(term: Term): Option[(Term, Seq[BooleanLiteral])] = {
+    if (term.isConditionOfIf)
+      Some((term, Seq(True, False)))
```

Comment:
We should also filter out if the term is already a `true` or `false, as those are already mutated